src/unistd.rs

```diff
@@ -1754,6 +1754,40 @@ pub fn mkstemp<P: ?Sized + NixPath>(template: &P) -> Result<(RawFd, PathBuf)> {
     Ok((fd, PathBuf::from(pathname)))
 }
 
+/// Creates a directory which persists even after process termination
+///
+/// * `template`: a path whose 6 rightmost characters must be X, e.g. `/tmp/tmpdir_XXXXXX`
+/// * returns: filename
+///
+/// Err is returned either if no temporary filename could be created or the template doesn't
+/// end with XXXXXX
+///
+/// See also [mkstemp(2)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/mkdtemp.html)
+///
+/// ```rust
```

Comment:
The "rust" part is unnecessary.  Rustdoc is smart enough to figure that out ;)

Reply:
Same as for mkstemp, I just tried to match it. If you think this and and the inline marker are useless, maybe they should all be removed in a separate commit

Reply:
Yes, the should be removed.  But don't introduce new ones.